homeassistant/components/group/__init__.py

```diff
@@ -381,13 +380,9 @@ def _async_process_config(hass, config, component):
 
         # Don't create tasks and await them all. The order is important as
         # groups get a number based on creation order.
-        group = yield from Group.async_create_group(
+        yield from Group.async_create_group(
```

Comment:
async_create_group already adds the group. Old code was adding each group twice.